### ISSUE TITLE
paddle2.0.1训练窗口日志参数哪里有详细介绍吗

### ISSUE THREAD

**Issue by futureflsl** [PaddlePaddle/PaddleDetection#2800]
第一次用这个，不知道这些参数含义，还有看什么参数可以停止训练，哪个loss呢
[04/28 17:21:56] ppdet.engine INFO: Epoch: [21] [120/141] learning_rate: 0.000770 loss_xy: 2.849720 loss_wh: 1.324514 loss_obj: 8.690937 loss_cls: 0.525484 loss: 14.034956 eta: 1:32:21 batch_cost: 0.1700 data_cost: 0.0012 ips: 5.8828 images/s
[04/28 17:21:59] ppdet.engine INFO: Epoch: [21] [140/141] learning_rate: 0.000775 loss_xy: 2.995280 loss_wh: 1.634839 loss_obj: 8.705307 loss_cls: 1.031295 loss: 14.335938 eta: 1:32:19 batch_cost: 0.1633 data_cost: 0.0010 ips: 6.1255 images/s
[04/28 17:22:00] ppdet.engine INFO: Epoch: [22] [  0/141] learning_rate: 0.000775 loss_xy: 3.105111 loss_wh: 1.848665 loss_obj: 9.094651 loss_cls: 1.226063 loss: 15.118809 eta: 1:32:20 batch_cost: 0.1690 data_cost: 0.0025 ips: 5.9185 images/s

**Comment by yghstill** [comment]
@futureflsl 
- `learning_rate `: 学习率
- `loss_xy `: 预测左上坐标loss
- `loss_wh `: 预测框宽高loss
- `loss_obj `: object loss
- `loss_cls `: 分类loss
- `loss`: 总loss

一般看总loss收敛是否平稳，或者加上--eval边训练边eval，看mAP变化决定什么时候停止训练。
